policy.md

```diff
@@ -1,178 +1,116 @@
+
+The entry for each subordinate certificate has “Audits Same as Parent” and “CP/CPS Same as Parent” checkboxes. When those are checked, the details do not need to be duplicated from the parent certificate. However, the subordinate certificate must be specifically listed in the audit statements of the parent certificate.
+
+URLs for the following documents are required for each certificate, unless the exceptions below apply:
+* CP
+* CPS
 * Standard audit (WebTrust or ETSI)
-* Baseline Requirements (BR) audit (if the certificate is capable of issuing
-  TLS/SSL server certificates)
+* Baseline Requirements (BR) audit (if the certificate is capable of issuing TLS/SSL server certificates)
 * Extended Validation (EV) SSL and/or Code Signing audit (if applicable)
 
 Exceptions to providing audit information:
-
-* The SHA-256 fingerprint of the certificate is specifically listed as in scope 
-in the audit statements of the parent certificate, and the 
-"Audits Same as Parent" checkbox is checked; or
+* The SHA-256 fingerprint of the certificate is specifically listed as in scope in the audit statements of the parent certificate, and the “Audits Same as Parent” checkbox is checked; or
 * The certificate has expired; or
-* The certificate is technically-constrained as described in section 7.1.5 
-of the CA/Browser Forum Baseline Requirements, or
-* The certificate has been revoked, and the corresponding record in the CCADB 
-has been updated with the correct revocation status.
-
-CAs must provide English versions of any Certificate Policy, Certification
-Practice Statement and Audit documents which are not originally in English,
-with version numbers matching the document they are a translation of. The
-English version is not required to be authoritative in all cases of dispute,
-but the CA must attest that the translation is not materially different to the
-original.
+* The certificate is technically-constrained as described in section 7.1.5 of the CA/Browser Forum Baseline Requirements, or
+* The certificate has been revoked, and the corresponding record in the CCADB has been updated with the correct revocation status.
+
+CAs must provide English versions of any CP, CPS and Audit documents which are not originally in English, with version numbers matching the document they are a translation of. The English version is not required to be authoritative in all cases of dispute, but the CA must attest that the translation is not materially different to the original.
+
+CAs must submit an [Add/Update Root Request Case](https://docs.google.com/document/d/1ttmeeqO6WxDWe_deDNsGUgDO_LpsvoduFNZeHHMw_f8/edit) to add or update audit information stored in the CCADB.
```

Comment:
That is true for root certificate records, but not for intermediate certificate records.
For intermediate certificate records, the CA must update the information directly.

Reply:
Added language to distinguish between the two, but need to resolve open comment about "subordinate" versus "intermediate" to resolve this conversation.